app/pages/error/template.test.js

```diff
@@ -7,46 +7,41 @@ const setup = {
 };
 
 describe('error page', () => {
-  it('should render the error title', componentTester(setup, (harness) => {
-    const context = { message: 'an error message' };
+  it('should render the error backLink', componentTester(setup, (harness) => {
+    const context = {
+      backLinkText: 'Error backLinkText',
+      backLinkHref: 'http://errorBackLinkHref.com',
+    };
```

Comment:
do we need tests for the default, ie when the context is not given?

Reply:
I don't think so, what if we changed them in BCL? We'd have to change them in all the apps